### ISSUE TITLE
Textarea text position

### ISSUE THREAD

**Issue by DirtyF** [Kinto/formbuilder#150]
Don't know why the second question field has an `height:auto` textarea who put too much space between the text and the focus:

![textarea-height](https://cloud.githubusercontent.com/assets/103008/18681093/c6df94f6-7f66-11e6-92bb-95f4b20bfa6c.png)

**Comment by almet** [comment]
I don't understand what you're proposing here? Isn't the created form already focusing different parts of the form depending on where the user is?

**Comment by DirtyF** [comment]
If you mean the very light gray background, I guess it is. If it does not bother anybody else but me, feel free to ignore this.

**Comment by rlecellier** [comment]
It seems linked with the idea of adding a marker to show that the questions are sortable. Generaly do something tell the user that each question is a item.

**Comment by almet** [comment]
I'm sorry @dirtyf I just don't understand what you are proposing (I'm not opposed to the idea). Can you rephrase?

**Comment by DirtyF** [comment]
@almet Sorry if I wasn't clear. I was just pointing that when a field is selected and active, a user should be able to distinguish it clearly from the others. The active zone should stand out, to help user focus and get a better visual confirmation of what he is edting. Could be a long select, with options, etc.

It's just a matter of showing what zone is active, and not just rely on the focus of the text field in the case of the screenshot. The whole field box is not very visible by default as it only relies on a very light background.

As a comparison other services like Google, Wufoo of Typeform clearly show what field is currently selected:

![google-field-focus](https://cloud.githubusercontent.com/assets/103008/18720010/1ec55110-802a-11e6-8edb-63d5d114e787.png)

![wufoo-field-focus](https://cloud.githubusercontent.com/assets/103008/18720011/1ec8a932-802a-11e6-96a1-f1d403f631c5.png)

![typeform-field-focus](https://cloud.githubusercontent.com/assets/103008/18720012/1ecfe170-802a-11e6-9ad4-27a0f231be55.png)

My 2 cents.

**Comment by almet** [comment]
Thanks for the clarification! We probably can make this easier to understand yes ! I like how Google does it by having a visible color mark.
